sklearn/feature_extraction/tests/test_text.py

```diff
@@ -832,6 +832,20 @@ def test_vectorizer_min_df():
     assert len(vect.stop_words_) == 5
 
 
+@pytest.mark.parametrize(
+    "min_df, max_df, message",
```

Comment:
instead of parametrizing on 2 variables, you can pass directly a dictionary with all the values